pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -67,11 +67,9 @@ type Meta struct {
 }
 
 type MetaSearchParams struct {
-	TenantID       string
-	MinFingerprint uint64
-	MaxFingerprint uint64
-	StartTimestamp int64
-	EndTimestamp   int64
+	TenantID                       string
+	MinFingerprint, MaxFingerprint model.Fingerprint
```

Comment:
I think it's good to use `model.Fingerprint` here instead of `uint64`

If you are changing `fp` here, maybe change above in `Ref` struct too?

Reply:
Yes, we can change it in the Ref as well. I would create a separate PR for that, though.